content/pt-br/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Limit Ranges
```

Comment:
It would be better to translate "Limit Ranges" in here too!

Reply:
Done!